src/components/dropdown-filter-ajax/dropdown-filter-ajax.js

```diff
@@ -126,6 +126,24 @@ class DropdownFilterAjax extends DropdownFilter {
      */
     create: PropTypes.func,
 
+    /**
+     * A callback function used to format the Ajax
+     * response into the format required by the table
+     *
+     * @property formatResponse
+     * @type {Function}
```

Comment:
Here it's probably quire important to annotate what the return value should be for this method, as the component expects quite strict value here - it has to be object with just following fields: 
{ 
`records` - number of items returned 
`items` - array of items in a format `{ id: ..., name: ... }`
`page` - current page number
}
Otherwise there will be error, so we should maybe even consider asserting on the return value and checking it contains all required fields or issuing warning in console? What's our approach here @andrewjtait ?